packages/eslint-plugin-office-addins/src/rules/load-object-before-read.ts

```diff
@@ -0,0 +1,292 @@
+      "getIntersectionOrNullObject",
+      "getLastCell",
+      "getLastColumn",
+      "getLastRow",
+      "getMergedAreas",
+      "getOffsetRange",
+      "getPivotTables",
+      "getRangeEdge",
+      "getResizedRange",
+      "getRow",
+      "getRowProperties",
+      "getRowsAbove",
+      "getRowsBelow",
+      "getSpecialCells",
+      "getSpecialCellsOrNullObject",
+      "getSpillParent",
+      "getSpillParentOrNullObject",
+      "getSpillingToRange",
+      "getSpillingToRangeOrNullObject",
+      "getSurroundingRegion",
+      "getTables",
+      "getVisibleView",
+      "getEntireColumn",
+      "getEntireRow",
+      "getIntersection",
+      "getIntersectionOrNullObject",
+      "getOffsetRangeAreas",
+      "getSpecialCells",
+      "getSpecialCellsOrNullObject",
+      "getUsedRangeAreas",
+      "getUsedRangeAreasOrNullObject",
+      "getRangeAreasBySheet",
+      "getRangeAreasOrNullObjectBySheet",
+      "getItemAt",
+      "getText",
+      "getDataBodyRange",
+      "GetHeaderRowRange",
+      "getSubstring",
+      "getTotalRowRange",
+      "getInvalidCells",
+      "getInvalidCellsOrNullObject",
+      "getDimensionValues",
+      "getByNamespace",
+      "getXml",
+      "getColumnLabelRange",
+      "getDataHierarchy",
+      "getFilterAxisRange",
+      "getPivotItems",
+      "getRowLabelRange",
+      "getDefault",
+      "getPrintArea",
+      "getPrintAreaOrNullObject",
+      "getPrintTitleColumns",
+      "getPrintTitleColumnsOrNullObject",
+      "getPrintTitleRows",
+      "getPrintTitleRowsOrNullObject",
+      "getParentComment",
+      "getAsImage",
+      "getActivePage",
+      "GetStencilInfo",
+      "getBase64Image",
+      "getHtml",
+      "getParagraphInfo",
+      "getByTitle",
+      "getRestApiId",
+      "getByName",
+      "getWindowSize",
+      "getActiveSectionOrNull",
+      "getActiveSection",
+      "getActiveParagraphOrNull",
+      "getActiveParagraph",
+      "getActivePageOrNull",
+      "getActivePage",
+      "getActiveOutlineOrNull",
+      "getActiveOutline",
+      "getActiveNotebookOrNull",
+      "getActiveNotebook",
+      "getCellPadding",
+      "getBorder",
+      "getParagraphBefore",
+      "getParagraphBeforeOrNullObject",
+      "getParagraphAfterOrNullObject",
+      "getParagraphAfter",
+      "getCellOrNullObject",
+      "getHeader",
+      "getFooter",
+      "getTextRanges",
+      "getOoxml",
+      "getNextTextRangeOrNullObject",
+      "getNextTextRange",
+      "getHyperlinkRanges",
+      "getLastOrNullObject",
+      "getDescendants",
+      "getAncestorOrNullObject",
+      "getAncestor",
+      "getByIdOrNullObject",
+      "getById",
+      "getLevelString",
+      "getLevelParagraphs",
+      "getBase64ImageSrc",
+      "getSelection",
+      "getByTypes",
+      "getByTag",
+      "getReferenceId", 
+      "getDocument",
+    ]);
+
+    function getPropertyThatHadToBeLoaded(node: TSESTree.Node | undefined): string | undefined {
```

Comment:
Currently this method doesn't appear to have anything to do with a property from a get statement like it's name suggests.  I know that is what you are using it for, but there is no tie or check for the get method so the name doesn't really fit.

Reply:
Ow, I guess I chose a confusing name
The `get` here really has nothing to do with the other gets.
The `get` here was more with the meaning: getSomeValue. I will change the function name because it got confusing.